pkg/sfu/downtrack.go

```diff
@@ -1935,7 +2012,7 @@ func (d *DownTrack) DebugInfo() map[string]interface{} {
 		"StreamID":            d.params.StreamID,
 		"SSRC":                d.ssrc,
 		"MimeType":            d.codec.MimeType,
-		"Bound":               d.bound.Load(),
+		"Bound":               d.bindState.Load(),
```

Comment:
Can just change this to `BindState` and make a `String()` method for `bindState` to make it more readable?